ui/app/clinical/constants.js

```diff
@@ -69,6 +69,8 @@ Bahmni.Clinical.Constants = (function () {
         concepts: concepts,
         otherActiveDrugOrders: "Other Active DrugOrders",
         dispensePrivilege: "bahmni:clinical:dispense",
+        prescribeMedicationPrivilege: "app:clinical:prescribeMedication",
```

Comment:
@vikramdasvaishnav-thoughtworks can we change this to "bahmni:clinical:" instead of "app:clinical" to make to consistent with dispense Privilege